node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
How did this get committed? (`node_modules` should be ignored)

Reply:
Not sure, Interesting.